libparsec/crates/types/src/account.rs

```diff
@@ -62,6 +63,14 @@ pub enum PasswordAlgorithm {
     },
 }
 
+/*
+*  VaultKeyAccess
+*/
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct VaultKeyAccess {
```

Comment:
`VaultKeyAccess` is going to be serialized, so it should be defined as a json5 in `types/schema/` following the RFC1014 specs